metrics/resource_view_test.go

```diff
@@ -74,6 +75,8 @@ type testExporter struct {
 	id string
 }
 
+func (testExporter) ExportView(viewData *view.Data) {}
```

Comment:
Out of interest: what is this needed for? I quickly checked the branch out and the tests still seem to pass with this line deleted

Reply:
Not all tests pass when using -tagnostackdriver. When a view is delete it exports the data before it is unregistered and if there is no method it will fail with a nill pointer exception.